src/Servers/Kestrel/Core/src/Internal/Http/HttpVersion.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http
 {
-    public enum HttpVersion
+    public enum HttpVersion : sbyte
```

Comment:
?

Reply:
`HttpVersionAndMethod` contains 3 parts
```csharp
int MethodEnd { get; set; }
HttpVersion Version { get; }
HttpMethod Method { get; }
```
To fit in a long `HttpMethod` can't be an `int`

Reply:
Any api changes are all to `.Internal` ones